model_compression_toolkit/core/common/fusion/fusing_info.py

```diff
@@ -198,9 +198,9 @@ def get_fused_nodes(self, op_id: str) -> Optional[List['BaseNode']]:
         """
         return self.fusing_data.get(op_id)
 
-    def get_nodes_to_disable_activation_quantization(self) -> List['BaseNode']:
+    def get_inner_fln_nodes(self) -> List['BaseNode']:
         """
-        Returns a list of the nodes that their activation quantization is disabled due to fusing.
+        Returns a list of the nodes that are part but not the last node of an FLN.
```

Comment:
seems that get_all_fused_operations returns a dict, which might not be ordered, and the we count on the order to remove the last node. isn't it a vulnerability?

Reply:
I think you're right.
Anyway, this is not related to this PR, I'm adding a TODO and I'll talk with @reuvenperetz about it to make sure.